src/ui/repo-item-delegate.cpp

```diff
@@ -218,6 +222,27 @@ void RepoItemDelegate::paintRepoItem(QPainter *painter,
                       &repo_desc_rect);
     painter->restore();
 
+    // Paint repo sharing owner for private share
+    if (static_cast<RepoCategoryItem*>(item->parent())->categoryIndex() ==
```

Comment:
Yeah, but we want to show the owner precisely in the private shared category, not outside.